app/controllers/hyku/registrations_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Hyku
+  class RegistrationsController < Devise::RegistrationsController
+    def new
+      return super if Settings.devise.account_signup
+      flash[:alert] = t(:'hyku.account_signup')
```

Comment:
Can we change this key to be something more descriptive.  `hyku.account.signup_unavailable` or similar?